packages/rest-generator/test/test-services/PriceCalculation/open-api/base.ts

```diff
@@ -0,0 +1,84 @@
+/**
```

Comment:
Question: What about the folders/package for model and api for better readability since we set this options in the generator/

Reply:
I tried these two options with vanilla open api command without templates, still didn't work.
Follow up tickets:
https://github.wdf.sap.corp/MA/scp-rocket/issues/1524